src/Namotion.Reflection/Context/ContextualType.cs

```diff
@@ -230,6 +239,76 @@ public IEnumerable<T> GetAttributes<T>()
             return ContextAttributes.OfType<T>().Concat(TypeAttributes.OfType<T>());
         }
 
+        /// <summary>
+        /// Gets a contextual property of the given contextual type (preserving the context).
+        /// </summary>
+        /// <param name="propertyName">The property name.</param>
+        /// <returns>The contextual property or null.</returns>
+        public ContextualPropertyInfo? GetProperty(string propertyName)
```

Comment:
Rename to GetContextualProperty()? Or is it too long and clear because you're calling this on a contextual type?